x/launch/types/request_content_test.go

```diff
@@ -125,5 +125,333 @@ func TestVestedAccountCodec(t *testing.T) {
 	require.NoError(t, err)
 	_, err = request.UnpackVestedAccount(cdc)
 	require.Error(t, err)
-	require.Equal(t, "not a vestedAccount request", err.Error())
+}
+
+func TestAccountRemoval_Validate(t *testing.T) {
+	tests := []struct {
+		name    string
+		content types.AccountRemoval
+		wantErr bool
+	}{
+		{
+			name: "invalid address",
+			content: types.AccountRemoval{
+				Address: "invalid_address",
+			},
+			wantErr: true,
+		}, {
+			name: "valid content",
+			content: types.AccountRemoval{
+				Address: sample.AccAddress(),
+			},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			err := tt.content.Validate()
+			if tt.wantErr {
+				require.Error(t, err)
+				return
+			}
+			require.NoError(t, err)
+		})
+	}
+}
+
+func TestGenesisAccount_Validate(t *testing.T) {
```

Comment:
I will not list all of them but several time we talk about "message" in error description like "valid message"

We should rename message to request